OpenRA.Game/Server/Server.cs

```diff
@@ -20,106 +20,47 @@
 using OpenRA.Primitives;
 using OpenRA.Support;
 
-using XTimer = System.Timers.Timer;
-
 namespace OpenRA.Server
 {
-	public enum ServerState : int
+	public enum ServerState
 	{
-		WaitingPlayers = 1,
-		GameStarted = 2,
-		ShuttingDown = 3
+		WaitingPlayers, GameStarted, ShuttingDown
 	}
 
 	public class Server
 	{
+		#region Members
+		
+		public readonly ModData ModData;
+		public readonly ServerSettings Settings;
+		public readonly Session LobbyInfo;
 		public readonly IPAddress Ip;
 		public readonly int Port;
-
-		int randomSeed;
 		public readonly MersenneTwister Random = new MersenneTwister();
+		public readonly List<string> TempBans = new List<string>();
 
-		// Valid player connections
-		public List<Connection> Conns = new List<Connection>();
+		// Valid and pre-verified player connections
+		readonly List<Connection> conns = new List<Connection>();
+		readonly List<Connection> preConns = new List<Connection>();
+		readonly TypeDictionary serverTraits = new TypeDictionary();
+		readonly TcpListener listener;
 
-		// Pre-verified player connections
-		public List<Connection> PreConns = new List<Connection>();
```

Comment:
I think anyone smart enough to be able to read would be able to infer that the pre in preConns is the same as "pre" in the comment one line above it, but whatever I'll change it back.